### PR TITLE
package: contracts-governance

### PR DESCRIPTION
**Pull request by tynes**
**Description**
Adds the governance contracts to the monorepo. This will let me import `GovernanceToken` into the devnet genesis config as a predeploy

### CODE REVIEW

**Review by mslipper — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -178,6 +178,24 @@ jobs:
...
+  contracts-governance-tests:
+    docker:
+      - image: ethereumoptimism/js-builder:latest
+    resource_class: xlarge
+    steps:
+      - restore_cache:
+          keys:
+            - v2-cache-yarn-build-{{ .Revision }}
+      - checkout
+      - run:
```
Comment (smartcontracts) [inline .circleci/config.yml:190]: Should we also add slither?

File: packages/contracts-governance/.nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v14.18.1
```
Comment (smartcontracts) [inline packages/contracts-governance/.nvmrc:1]: Included at root level

**Review by smartcontracts — APPROVED** [review]
Code itself could use some cleanup but not really here to address that. Let's get it merged then do cleanup PRs as we go.

File: packages/contracts-governance/package.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  "files": [
+    "dist/**/*.js",
+    "dist/**/*.d.ts",
+    "dist/types/*.ts",
+    "artifacts/**/*.json",
+    "deployments/**/*.json"
+  ],
+  "scripts": {
+    "build": "npx hardhat compile",
+    "test": "npx hardhat test",
```
Comment (maurelian) [inline packages/contracts-governance/package.json:17]: Any reason to use `npx` here but `yarn solhint` below?

Comment (tynes) [reply packages/contracts-governance/package.json:17]: this was copy pasted from the other repo, can clean up

File: .circleci/config.yml (hunk 1 above)
Comment (tynes) [reply .circleci/config.yml:190]: slither isn't configured for this package and I don't feel like setting it up, not much dev will happen here and we could consider consolidating this package into others in the future

**Pull request merged**